src/docs/implementations/dynatrace.adoc

```diff
@@ -219,7 +224,10 @@ These properties can also be set in Spring Boot configuration files:
 
 [source,yml]
 ----
+# for Spring Boot 2.x.x
 management.metrics.export.dynatrace:
+# When using Spring Boot 3.0.0 or above, use the line below instead of the line above.
+# management.dynatrace.metrics.export:
```

Comment:
Shouldn't we default to Spring Boot 3 here?

Reply:
✅